Source/Core/Core/IOS/Network/IP/Top.h

```diff
@@ -13,6 +13,13 @@
 #include <ws2tcpip.h>
 #endif
 
+// WSAPoll doesn't support POLLPRI and POLLWRBAND flags
+#ifdef _WIN32
+#define UNSUPPORTED_WSAPOLL POLLPRI | POLLWRBAND
```

Comment:
Sorry, that was an oversight of #3316. It was missing parentheses.

Can you please change it to a `constexpr` constant instead
```C++
constexpr int UNSUPPORTED_WSAPOLL = POLLPRI | POLLWRBAND;
```